fine_tuning_sql/notebooks/2_data_preparation.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 2: Generate training files and train the model in SambaStudio\n",
+    "\n",
+    "This notebook is a guide of how to convert your jsonl files to hdf5 files that are the files you will need to upload in order to start a training job in Sambastudio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "current_dir = os.getcwd()\n",
+    "kit_dir = os.path.abspath(os.path.join(current_dir, \"..\"))\n",
+    "repo_dir = os.path.abspath(os.path.join(kit_dir, \"..\"))\n",
+    "\n",
+    "sys.path.append(kit_dir)\n",
+    "sys.path.append(repo_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generative data preparation package dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The [generative Data preparation package]() is a package developed by Sambanova systems used to prepare your data to be used as dataset in your SambaStudio Environment, this package is included as git submodule in the AI Starter kit, so for instalations you should onlY execute teh following line"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "generative_data_prep_dir = os.path.join(repo_dir, \"utils\", \"fine_tuning\", \"generative_data_prep\")\n",
+    "! pip install $generative_data_prep_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generative data preparation usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For Dataset generation you will need to specify the tokenizer of the model you want to train setting its model id in the `TOKENIZER` variable\n",
+    "- [Llama 2](https://huggingface.co/meta-llama/Llama-2-7b-hf): meta-llama/Llama-2-7b-hf\n",
+    "- [Llama 3](https://huggingface.co/meta-llama/Meta-Llama-3-8B): meta-llama/Meta-Llama-3-8B\n",
+    "- [Mixtral](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2): mistralai/Mistral-7B-Instruct-v0.2\n",
```

Comment:
Lets get rid of Mixtral as we don't support this model yet

Reply:
it was a typo,  it meant to be mistral 7b